### PR TITLE
Update README.md adding CVT to the CAF models table.

### PR DESCRIPTION
**Pull request by DanielJacksonTfN**
# Changes

Added CVT to the CAF models table.

# Tasks

- [ ] Have unittests been added (testing individual functions and their edge cases)
- [ ] Has documentation (including user guide) been updated
- [ ] Have new dependencies been added (or changed) in relevant `requirements.txt`
- [ ] Code linting, tests and other workflows pass

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -118,6 +118,7 @@ List of some of the key models available under CAF.
...
 | [caf.carbon](https://github.com/Transport-for-the-North/caf.carbon/)               | Bespoke carbon emissions model to forecast travel fleet data and their emissions.                                                                                                                                                                                       | Carbon modelling        | Code, CLI      | Release | Python   |
+| [caf.cvt](https://github.com/Transport-for-the-North/caf.cvt/)               | Climate Vulnerability model to assign climate hazard risks to the transport network.                                                                                                                                                                                       | Climate modelling        | Code, CLI      | Release | Python   |
```
Comment (MattBuckley-TfN) [inline profile/README.md]: I think I'd probably put the status as Beta to match the status in the repo's README but other than that it looks good.

**Review by MattBuckley-TfN — CHANGES_REQUESTED** [review]

**Review by MattBuckley-TfN — APPROVED** [review]
LGTM
